src/index.scss

```diff
@@ -0,0 +1,19 @@
+@import '../node_modules/normalize.css/normalize.css';
+@import '../node_modules/font-awesome/scss/font-awesome.scss';
+@import './components/CategoriesContainer.scss';
```

Comment:
You don't need to connect components to the `index.scss`, `sass-loader` gets to do that instead of you =)

Reply:
And because CSS is Cascading Style Sheets in `index.scss` you need to keep the order of connecting items